app/vmagent/remotewrite/client.go

```diff
@@ -456,10 +456,9 @@ again:
 
 	// Unexpected status code returned
 	retriesCount++
-	retryDuration *= 2
-	if retryDuration > maxRetryDuration {
-		retryDuration = maxRetryDuration
-	}
+	retryDuration = calculateRetryDuration(parseRetryAfterHeader(resp.Header.Get("Retry-After")), retryDuration, maxRetryDuration)
```

Comment:
I think we should log whether we got Retry-After header, and log if we got it but failed to parse.

Reply:
How about logging them both with 1 line:
```
2024-09-04T03:34:02.237Z	info	app/vmagent/remotewrite/client.go:520	found Retry-After header 10, parsed into 10.00 second(s)
2024-09-04T03:34:02.237Z	info	app/vmagent/remotewrite/client.go:520	found Retry-After header Wed, 04 Sep 2024 03:34:32 GMT, parsed into 29.00 second(s)
2024-09-04T03:34:02.237Z	info	app/vmagent/remotewrite/client.go:520	found Retry-After header invalid-retry-after, parsed into 0.00 second(s)
2024-09-04T03:34:02.237Z	info	app/vmagent/remotewrite/client.go:520	found Retry-After header Wed, 04 Sep 2024 11:34:12 FAKETZ, parsed into 0.00 second(s)
```

It should be simple enough for user to know whether it's parsed correctly.